app/src/main/java/com/example/drawingApp/Hsv.kt

```diff
@@ -0,0 +1,31 @@
+package com.example.drawingApp
+
+import android.graphics.Color
+
+data class Hsv(
+    val hue: Float? = null,
```

Comment:
Why make these nullable? You don't need to return a nullable color if these aren't nullable. If any of these would be null, make the whole object null, not one of the fields. It seems odd to have an object that represents a color but then someone wouldn't provide all the properties of the color fields HSV.